styles.css

```diff
@@ -0,0 +1,232 @@
+* {
```

Comment:
you should add files generated by tools to gitignore. As you are using `scss`, there is no need to push to repo generated files (every dev can generate them by itself, using scss tool). 

Please, remove all `.css` files, commit. Add to gitignore all css files and commit once again after generating them by scss tool. They should disappear from githube repo after push ;)